libafl/src/mutators/mutations.rs

```diff
@@ -1200,6 +1194,153 @@ impl<I> CrossoverReplaceMutator<I> {
         }
     }
 }
+/// Crossover insert mutation for inputs mapped to a bytes vector
+#[derive(Debug)]
+pub struct MappedCrossoverInsertMutator<'a, F> {
+    input_from_corpus_mapper: &'a F,
+}
+
+impl<'a, S, F, I> Mutator<I, S> for MappedCrossoverInsertMutator<'a, F>
+where
+    S: HasCorpus + HasMaxSize + HasRand + UsesInput,
+    I: HasMutatorBytes,
+    F: for<'b> Fn(&'b S::Input) -> Option<&'b [u8]>,
```

Comment:
Why does this F return a `[u8]` and not a `WrappedInput`?

Reply:
Because `WrappedInput` implementations generally cannot be used with immutable references. You only get an immutable reference from the input chosen from the corpus, so you can't create a `MutVecInput` or something similar.